reactor-client/pom.xml

```diff
@@ -6,7 +6,7 @@
     <parent>
         <groupId>com.aerospike</groupId>
         <artifactId>aerospike-client-java-reactive</artifactId>
-        <version>5.0.7</version>
+        <version>5.1.0</version>
```

Comment:
We use the Java client version in releases. So this one should be `5.1.8`.

Reply:
Done.